sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/GetBoundingBoxes.java

```diff
@@ -70,8 +70,8 @@ public static void main(String[] args) {
                             System.out.printf("Cell text %s has following words: %n", formTableCell.getText());
                             // FormElements only exists if you set includeFieldElements to true in your
                             // call to beginRecognizeCustomFormsFromUrl
-                            // It is also a list of FormWords and FormLines, but in this example, we only deal with
-                            // FormWords
+                            // It is also a list of FormWords, FormLines and FormSelectionMarks, but in this example,
+                            // we only deal with FormWords.
```

Comment:
why not showcase selection marks in this sample as well?

Reply:
I see, you created a new sample for that

Reply:
Yeah. A sample, RecognizeCustomFormsAsyncWithSelectionMarks